views/index.erb

```diff
@@ -5,27 +5,49 @@
   <%= erb :header %>
 
   <script type="text/javascript">
-    tours['showoff:activity'] = [
-      {
-        element: ".currentSlide .activityToggle",
-        intro: I18n.t('tour.activity.complete')
-      }
-    ];
-
-    tours['showoff:form'] = [
-      {
-        element: ".currentSlide input.save",
-        intro: I18n.t('tour.form.save')
-      }
-    ];
-
-    tours['showoff:menu'] = [
-      {
-        element: "#hamburger",
-        intro: I18n.t('tour.menu')
-      }
-    ];
-
+    $( document ).ready(function() {
```

Comment:
I missed that.  We really should be doing something to better identify the view in the code so that we don't have to do this in the views.  Or perhaps we need:
showoff.js
presenter.js
audience.js

I guess since we've already gone down this path, it can wait for a rewrite later.